proof/proof/inc/TProofOutputList.h

```diff
@@ -30,7 +30,6 @@ class TProofOutputList : public TList {
 
 public:
    TProofOutputList(const char *dontshow = "PROOF_*");
-TProofOutputList(TObject *o) : TList(o), fDontShow(0) { } // for backward compatibility, don't use
```

Comment:
I am not sure that a comment of this constructor will save us from the possibility of legacy code actually using it. We might be lucky or we might not. Given that proof is not built by default anymore but it's still available, we probably want to at least document this in the release notes

Reply:
Right, it should be mentioned